### ISSUE TITLE
Ability to add custom nginx conf

### ISSUE THREAD

**Issue by bilyboy785** [Mailu/Mailu#534]
Hey, 

It's possible to add custom nginx conf for adding for example a /cloud for Nextcloud docker ?
I can't find a way to access Nginx conf on the mailu front.

Regards,

**Comment by ofthesun9** [comment]
You can download the nginx.conf embedded in Malilu:
Modify it to add your /cloud section and modify your docker-compose.yml to mount this file at /conf; in the front service:
      - "$ROOT/certs:/certs"
      - "$ROOT/nginx.conf.cloud:/conf/nginx.conf"

**Comment by bilyboy785** [comment]
Thank's ! But i'm blocking on start. I set variable : 

```
      set $admin {{ HOST_ADMIN }};
      set $antispam {{ HOST_ANTISPAM }};
      set $webmail {{ HOST_WEBMAIL }};
      set $webdav {{ HOST_WEBDAV }};
      set $nextcloud {{ HOST_NEXTCLOUD }};
```

And my location : 

```
location nextcloud {
        rewrite ^(nextcloud)$ $1/ permanent;
        rewrite ^nextcloud/(.*) /$1 break;
        include /etc/nginx/proxy.conf;
        client_max_body_size {{ MESSAGE_SIZE_LIMIT|int + 8388608 }};
        proxy_pass http://$nextcloud;
      }
```

On start, nginx warn me : 

```
nginx: [emerg] invalid number of arguments in "set" directive in /etc/nginx/nginx.conf:36
```

I see the config.py file set vars HOST_... : 

https://github.com/Mailu/Mailu/blob/master/core/nginx/config.py

Do i miss something ?

**Comment by kaiyou** [comment]
Indeed, if you are willing to mount the nginx config file, please keep in mind that it is compiled first. So you should mount to ``/conf/nginx.conf`` if you want your file to get compiled :)

**Comment by muhlemmer** [comment]
Should this be included in FAQ? #564.

**Comment by rakshith-ravi** [comment]
@muhlemmer yes I believe this question is pretty common. A lot of people tend to have their own front container instead of the default one. So I'd recommend that this be added to the FAQ.

If possible, please also document how the users can expose the required ports in other containers and make their own front container instead of the default one, since most people tend to add the different services to subdomains instead of paths.

Ex: people should be able to read the FAQ, remove the front container from the docker-compose and create their own nginx config to perhaps have mail.example.com, mailadmin.example.com antivirus.example.com, etc. I hope I've made my point clear?
